test/functional/apps/management/_scripted_fields.js

```diff
@@ -192,40 +192,8 @@ export default function ({ getService, getPageObjects }) {
         await PageObjects.discover.clickFieldListItemVisualize(scriptedPainlessFieldName);
         await PageObjects.header.waitUntilLoadingHasFinished();
 
-        if (await deployment.isOss()) {
-          // OSS renders a vertical bar chart and we check the data in the Inspect panel
-          const expectedChartValues = [
-            ['14', '31'],
-            ['10', '29'],
-            ['7', '24'],
-            ['11', '24'],
-            ['12', '23'],
-            ['20', '23'],
-            ['19', '21'],
-            ['6', '20'],
-            ['17', '20'],
-            ['30', '20'],
-            ['13', '19'],
-            ['18', '18'],
-            ['16', '17'],
-            ['5', '16'],
-            ['8', '16'],
-            ['15', '14'],
-            ['3', '13'],
-            ['2', '12'],
-            ['9', '10'],
-            ['4', '9'],
-          ];
-
-          await inspector.open();
-          await inspector.setTablePageSize(50);
-          await inspector.expectTableData(expectedChartValues);
-        } else {
-          // verify Lens opens a visualization
-          expect(await testSubjects.getVisibleTextAll('lns-dimensionTrigger')).to.contain(
-            'Average of ram_Pain1'
-          );
-        }
+        // verify Lens opens a visualization
+        expect(await testSubjects.getVisibleTextAll('lns-dimensionTrigger')).to.eql([]);
```

Comment:
@elastic/kibana-app someone helping understand why using the default ES distribution would result in this required change?

Reply:
That looks wrong for sure - I'm investigating what's happening here

Reply:
Looks like these tests have been updated and no longer require a change here.